unit-tests/dds/d435i.py

```diff
@@ -172,198 +172,198 @@ def ir_stream( number ):
 
 def color_stream_profiles():
     return [
-        dds.video_stream_profile( 30, dds.stream_format("rgb8"), 1920,1080 ),
+        # dds.video_stream_profile( 30, dds.stream_format("rgb8"), 1920,1080 ),
         dds.video_stream_profile( 30, dds.stream_format("BYR2"), 1920,1080 ),
```

Comment:
Why leave `BYR2` and not YUYV?

Reply:
As discussed, D435i has a `RAW16` color profile.

Reply:
Then why are the rest of the profiles YUYV?

Reply:
I checked and this is indeed what's returned from Media Foundation (at least, for the D415 & D455) -- one profile of BYR2 at max res... (added debug code, might end up in a PR)